print_pdf() error -> PermissionError: [WinError 32] O arquivo já está sendo usado por outro processo

**Describe the bug**

Raises error when i use `print_pdf()` **only** in EDGE headful mode:
```python
def print_pdf(self, path=None, print_options=None):
    """Print the current page as a PDF file.
    
    Args:
        path (str, optional): The path for the file to be saved. Defaults to None.
        print_options (dict, optional): Print options as defined at. Defaults to None.
    
    Returns:
        str: the saved file path
    """
    title = self.page_title() or "document"
    timeout = 60000
    default_path = os.path.expanduser(os.path.join(self.download_folder_path, f"{title}.pdf"))
    
    if self.browser in [Browser.CHROME, Browser.EDGE] and not self.headless:
        pdf_current_count = self.get_file_count(file_extension=".pdf")
        # Chrome still does not support headless webdriver print
        # but Firefox does.
        self.execute_javascript("window.print();")
    
        # We need to wait for the file to be available in this case.
        if self.page_title():
            self.wait_for_file(default_path, timeout=timeout)
        else:
            # Waiting when the file don't have the page title in path
            self.wait_for_new_file(file_extension=".pdf", current_count=pdf_current_count)
    
        # Move the downloaded pdf file if the path is not None
        if path:
            last_downloaded_pdf = self.get_last_created_file(self.download_folder_path, ".pdf")
>           os.rename(last_downloaded_pdf, path)
E           PermissionError: [WinError 32] O arquivo já está sendo usado por outro processo: 'D:\\BotCity\\projetos\\dev\\web\\Botcity - web test.pdf' -> 'D:\\BotCity\\projetos\\dev\\web\\page.pdf'

venv\lib\site-packages\botcity\web\bot.py:992: PermissionError
```

**Expected behavior**
Save pdf and rename.

**Steps to Reproduce**
```python
web = WebBot()
web.headless = False
web.browser = Browser.EDGE
web.driver_path = os.path.join(project_path, 'web-drivers', 'msedgedriver.exe')

web.browse(os.path.join(project_path, 'web', 'index.html'))
pdf = web.print_pdf(path=os.path.join(project_path, 'page.pdf'))
print(pdf)
```

**My Platform**
    * OS: Windows 11
    * Python Version: 3.10.2
    * Packages Version: `botcity-framework-web==0.4.0`

--- a/botcity/web/bot.py
+++ b/botcity/web/bot.py
@@ -1890,6 +1890,7 @@ class WebBot(BaseBot):
                 if self.browser == Browser.FIREFOX and os.path.exists(path + '.part'):
                     # if *.part exists, the download is not completed.
                     continue
+                self.sleep(config.DEFAULT_SLEEP_AFTER_ACTION)
                 return True
             self.sleep(config.DEFAULT_SLEEP_AFTER_ACTION)
 
@@ -1948,7 +1949,8 @@ class WebBot(BaseBot):
             elapsed_time = (time.time() - start_time) * 1000
             if elapsed_time > timeout:
                 return None
-            pdf_count = self.get_file_count(path, f"*{file_extension}")
-            if pdf_count == current_count + 1:
+            file_count = self.get_file_count(path, f"*{file_extension}")
+            if file_count == current_count + 1:
+                self.sleep(config.DEFAULT_SLEEP_AFTER_ACTION)
                 return self.get_last_created_file(path, f"*{file_extension}")
             self.sleep(config.DEFAULT_SLEEP_AFTER_ACTION)

